proposal.md

```diff
@@ -68,14 +69,16 @@ code files provided to appropriate places, and creates skeleton help
 files and a Read-and-delete-me file describing further steps in
 packaging.
 
-Some contributed packages provide literate programming frameworks to construct 
-packages from code embedded in text documents, or frameworks for special types 
-of packages such as Shiny apps or metapackages.
+WRE reference: [Package Structure](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-structure).
 
-WRE reference: [Package
-Structure](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-structure).
+The `r pkg("usethis", priority = "core")` package has many utilities to setup the structure of the package, including a NEWS file and adding dependencies in the README.
+The `r pkg("pkgKitten")` package provides an almost empty package skeleton and some utilities to create help pages.
+The `r bioc("biocthis")` package prepares for Bioconductor repository.
 
-CONSIDER: usethis, biocthis, fusen, litr, noweb, rfold, many, DataPackageR
+Some packages like `r pkg("fusen")` and `r github("jacobbien/litr-project")` prepare a project from an R markdown file. 
+`r pkg("noweb")` allows to use literate programming to create the package.
+There are others that help packaging several R files, `r pkg("rfold")`, or create a package from a dataset `r pkg("DataPackageR")`.
+Others, like `r pkg("cookiecutter")` can create a new package from a template stored elsewhere. 
```

Comment:
Please use bullet points for the contributed packages/services, so the overall structure of each section is 

<opening comments, highlighting base R/official R project tools>
<list of contributed packages/services>

If necessary, you can use sectioning to provide further structure. 

Although existing task views do often have lists of the form

* The [foo]() package does this
* The [bar]() package does that

I think it is better to simplify to

* [foo]() does this
* [bar]() does that

unless you are talking about something that is not an R package.